server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -64,6 +65,13 @@ public class SniffConnectionStrategy extends RemoteConnectionStrategy {
         }, new StrategyValidator<>(ns, key, ConnectionStrategy.SNIFF), Setting.Property.Dynamic, Setting.Property.NodeScope)
     );
 
+    public static final Setting.AffixSetting<String> REMOTE_CLUSTER_AUTHORIZATION = Setting.affixKeySetting(
+        "cluster.remote.",
+        "authorization",
```

Comment:
You say in the PR description that the setting is `Authorization` and is case sensitive, but it's `authorization` (all lowercase) here. I think what's here currently is more closely aligned with our other settings, so can you update the PR description?

Reply:
I agree. I made the change.